### ISSUE TITLE
Draw only style updates without data updates

### ISSUE THREAD

**Issue by djbarnwal** [finos/regular-table#201]
## Support Question

Is there a way or an API to invoke only the style listeners without invoking the data listeners. The `table.draw` API updates both the data as well as style.

Sometimes to add more interactivity to the table (such as highlighting the column containing the cell on hover) requires recalculating the styles of the cells. Currently I achieve this by calling `table.draw` but this also invokes the data callback which adds an unnecessary overhead.

**Comment by texodus** [comment]
I don't think this can be done currently - I would be ok with adding this API but it is a medium complexity change as the component does not currently cache the data listener pages.

You may be able to achieve what you want by just styling the grid directly without calling `draw()` - e.g. `getMeta` calls on the `td` elements work outside of a style listener callback. You may also be able to workaround by memoizing `table.innerHTML` in the initial draw listener call, then swap this pre-styled copy back into the DOM in your hover handler.
